OpenGLViewSample/OpenGLViewSample/AppDelegate.cs

```diff
@@ -21,6 +21,7 @@ public AppDelegate ()
 		public override void DidFinishLaunching (NSNotification notification)
 		{
 			mainWindowController = new MainWindowController ();
+            ObjCRuntime.Class.ThrowOnInitFailure = false;
```

Comment:
Spaces not tabs.

Reply:
@ivanherreragl : Can you create a separate PR for Bundle Identifier fixes 
91ce8ec?
Commits  edcf54e , 5ce36d0, b6bc075 are already created as separate PRs and are under review.